web/src/features/map/Map.tsx

```diff
@@ -126,6 +128,15 @@ const Map = ({ children, zoom, center, renderTooltip }: Props) => {
     map.getView().setCenter(center)
   }, [center]) // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    if (!map) return
+
+    // Wait enough time for the side panel to completely collapse then resize the map
+    setTimeout(() => {
+      map.updateSize()
+    }, 700)
```

Comment:
Isn't there a better way to do this? I don't like relying on an arbitrary number. What happens on a slow computer where the panel takes 701ms?

Reply:
I'd also like to see a better way of doing this. I noticed that the animation of closing the panel was not very smooth, e.g. for a while there's a delay where half the screen is just blank, and then the map gets redrawn.

Reply:
It's slow because of all the graph rendering - and yeah it's going to be slower with slow browsers. I am thinking of removing the animation. That way when I set 0.1s for wait time, it will always work.